src/App.tsx

```diff
@@ -21,7 +21,7 @@ export const App: React.FC<{}> = () => (
               <HomePage />
             </Route>
             {ITProfessions.map((r) => (
-              <Route exact path={`/${r}`} key={r}>
+              <Route path={`/${r}`} key={r}>
```

Comment:
I'd be careful with removing the `exact` param here! :) The routes for professions will be different so we're safe for now, but we may (and probably should) decide to have a common prefix for profession routes (something like `/detail/${profession}`. In this case, we need `exact` to make sure we get to the right routes. More about it here: https://stackoverflow.com/a/49162423.

Reply:
To be honest I don't know how to make working tabs with Routing without removing exact param here.

Reply:
As it was just my silly idea to add routing to those tabs and I didn't know that it can be so problematic in React, I think we should create a new task to work on that issue in the next sprint :)